### DISCUSSION TITLE
Merging volumes and annotations

### DISCUSSION THREAD

**Discussion by BernardP437** [biigle/core#509] [Q&A] [answered]
We started a project to evaluate BIIGLE for shallow water imagery annotation. We created 4 volumes as we couldn't see how to add files to an existing volume on the BIIGLE storage. Is it possible to merge those volumes and their annotations? We forked the CATAMI label tree for our use. 

thanks, 
Bernard Picton

**Comment by mzur** [comment, marked as answer]
If this is a one-time thing, the volumes could be merged manually by us BIIGLE admins. Please write an email to info@biigle.de with the links to the volumes and I will have a look.

Generally, you can extend directories of uploaded files if you use the same directory name in the new storage request. For example, if you uploaded `my-images/1.jpg` and then in another storage request upload `my-images/2.jpg`, you can add `2.jpg` to the existing volume (on the volume edit page). The user interface of the volume edit page is one thing that we [would like to improve](https://github.com/biigle/core/issues/454) in this regard.

**Comment by BernardP437** [comment]
Hi Martin,

Thanks! Yes, this is one-off as we didn't work out how to add the second,
third and fourth batches of files. We did upload all the files then to
another volume but couldn't see how to then attach annotations already done
on those images. We created one directory and volume that we then didn't
use (MercReef2)

So there are three volumes/directories with annotations (and one without)
plus a directory with all the images (MercReefAll) but no annotations.

Also a label tree, CATAMI MercReefs

Project Merc2022

Volumes MercReef, MercReef3, MercReef4 are annotated

They are the files in directories MercReef, MercReef3, MercReef4 - your
reports come back as 1596 (project ID?) and in that 7838, 7953, 7986
annotation reports

all the best,

Bernard
